sdcm/argus_results.py

```diff
@@ -41,8 +43,9 @@ class Meta:
             ColumnMetadata(name="Throughput write", unit="op/s", type=ResultType.INTEGER, higher_is_better=True),
             ColumnMetadata(name="Throughput read", unit="op/s", type=ResultType.INTEGER, higher_is_better=True),
             ColumnMetadata(name="duration", unit="HH:MM:SS", type=ResultType.DURATION, higher_is_better=False),
+            ColumnMetadata(name="start time", unit="", type=ResultType.TEXT),
```

Comment:
Why don't we want this as a number ?

Do we assume we'll never plot or sort by this one ?

Reply:
no, it just serves investigation purposes to quickly get to proper log place (or looking at monitoring).

Reply:
maybe we should document this at some place in the code ?

Reply:
I added a comment line. Also found that rebasing have 'eaten' changes in other tables - fixed.